.github/workflows/build_and_test.yml

```diff
@@ -357,11 +357,7 @@ jobs:
         architecture: x64
     - name: Install Python linter dependencies
       run: |
-        # TODO(SPARK-32407): Sphinx 3.1+ does not correctly index nested classes.
```

Comment:
I would just keep them. I intentionally separated them for self-contained dependency list. 

Python linter also runs documentation build so we need the rest of dependencies

Reply:
okay. Let's close this issue. Thanks for the review.